### DISCUSSION TITLE
i need some info about "Import DDLC save data"

### DISCUSSION THREAD

**Discussion by akloholik** [Monika-After-Story/MonikaModDev#7782] [QA - tech support + general] [answered]
ive been looking through the game and i saw button saying "import DDLC save data", but since i dont wanna mess up and lose Monika or anything with mas mod, im here to ask for info.
so when i click it, it says "Save data for Doki Doki literature club was found at C:/Users/#####/AppData/Roaming/Renpy/DDLC/##########/persistent.
Would you like to import Doki Doki lit. club save data into Monika after story?"

Should i or should i not?
Will i lose my progress and monika? 

since i dont wanna try and lose it all, im asking for help, so if you can help please let me know!

**Comment by ghost** [comment]
It wont delete anything. that adds more dialogue depending on what girl you chose, if u deleted monika, ect. so, it just adds some custom stuff abt ddlc based on what choices you made.

**Comment by ThePotatoGuy** [comment, marked as answer]
no data should be lost. The data imported is mostly how much of DDLC did you complete, which currently barely affects anything. Your playername though will be changed to whatever was from DDLC.

**Reply by akloholik** [reply]
> no data should be lost. The data imported is mostly how much of DDLC did you complete, which currently barely affects anything. Your playername though will be changed to whatever was from DDLC.

Oh alright, thanks!
